en/flight_controller/cuav_v5_plus.md

```diff
@@ -145,32 +145,7 @@ The provided debug cable does not connect to the SWD port `Vref` pin (1).
 Any multicopter / airplane / rover or boat that can be controlled with normal RC servos or Futaba S-Bus servos.
 The complete set of supported configurations can be seen in the [Airframes Reference](../airframes/airframe_reference.md).
 
-
-## Known Issues
-
-The issues below refer to the *batch number* in which they first appear. 
-The batch number is the last two digits of the *serial number* that appears on a sticker on the side of the flight controller. 
-For example, the serial number V011907XXXX**01** indicates the flight controller was in batch 01.
-
-#### GPS not compatible with other devices (Critical)
-
-The *Neo v2.0 GPS* recommended for use with *CUAV V5+* and *CUAV V5 nano* is not fully compatible with other Pixhawk flight controllers (specifically, the buzzer part is not compatible and there may be issues with the safety switch).
-The GPS will not work with other flight controllers, and is the only GPS unit that can be used with the *CUAV V5+* and *CUAV V5 nano*.
-<!-- 5+/90/V5+ 20190523 RC01 -->
-
-- *Found:* Batch 01
-- *Fixed:* -
-
-#### Volt regulation varies greater than +/- 5%
```

Comment:
FYI as above, deleted because this is a feature: https://github.com/PX4/px4_user_guide/pull/698/files#r398207342